lib/components/visualization/tables/SortButton.js

```diff
@@ -63,8 +63,10 @@ var SortButton = React.createClass({
 	 * @return {jsx}
 	 */
 	render: function () {
+		let sort_icon = (this.state.direction === 'asc') ? 'icon-chevron-up' : 'icon-chevron-down';
```

Comment:
I believe that this can just be `this.state.direction === 'asc' ? ... : ...` (no parens needed)

Also, this might be a good place to use the `classNames` library:

```js
classes: {
  'icon-chevron-up': this.state.direction === 'asc',
  'icon-chevron-down': this.state.direction === 'desc',
  'pull-right': true,
}
```

(that might be 100% right syntax)